bin/cucumber

```diff
@@ -15,7 +15,7 @@ ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
 bundle_binstub = File.expand_path("bundle", __dir__)
 
 if File.file?(bundle_binstub)
-  if /This file was generated by Bundler/.match?(File.read(bundle_binstub, 300))
+  if File.read(bundle_binstub, 300).include?("This file was generated by Bundler")
```

Comment:
Perhaps we don't want to lint the binstub files generated by other gems?

Reply:
Yes, that's a good point.